dipy/io/stateful_tractogram.py

```diff
@@ -37,21 +34,6 @@ def set_sft_logger_level(log_level):
     logger.setLevel(level=log_level)
 
 
-class Space(enum.Enum):
-    """Enum to simplify future change to convention"""
-
-    VOX = "vox"
-    VOXMM = "voxmm"
-    RASMM = "rasmm"
-
-
-class Origin(enum.Enum):
-    """Enum to simplify future change to convention"""
-
-    NIFTI = "center"
-    TRACKVIS = "corner"
```

Comment:
a deprecation cycle is needed, this can not be moved like this. it will need 2 releases at least before complete deletion from this module

Reply:
Since stateful_tractogram.py still imports the class (from utils) the line `from dipy.io.stateful_tractogram import Space, Origin` still works like before.

Is the deprecation warning still relevant? (since it will still work after a future deprecation)